src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for a Person's details in the addressbook.
+ * Implementations should guarantee: details are present and not null, field values are validated.
+ */
+public interface Printable {
+    String getPrintableString();
```

Comment:
Missing header comment. All non-trivial classes and methods should have javadoc format header comments.